runelite-client/src/main/java/net/runelite/client/plugins/thieving/ThievingConfig.java

```diff
@@ -26,13 +26,28 @@
  */
 package net.runelite.client.plugins.thieving;
 
+import java.awt.Color;
+import net.runelite.client.config.Alpha;
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
+import net.runelite.client.config.Range;
 
 @ConfigGroup("thieving")
 public interface ThievingConfig extends Config
 {
+	@ConfigSection(
```

Comment:
Can you order these the way they usually are ordered? Ie the same order they go in the config tab.

Positions aren't shared between sections and config items, so both chest section and stat timeout can be in position 1

Reply:
Fixed